frontend/src/scenes/insights/insightSceneLogic.tsx

```diff
@@ -230,6 +230,11 @@ export const insightSceneLogic = kea<insightSceneLogicType>([
                     ? {
                           activity_scope: ActivityScope.INSIGHT,
                           activity_item_id: `${insight.id}`,
+                          // when e.g. constructing URLs for an insight we don't use the id,
+                          // so we also store the short id
+                          activity_item_context: {
+                              short_id: `${insight.short_id}`,
+                          },
```

Comment:
this will mean we can't open a page for comments on insights made before this, but it's marked as experimental in the UI so we can be a little lazy here